guide/kh/index.md

```diff
@@ -0,0 +1,5 @@
+---
+section: guide
+lang: KH
+title: Khmer
+---
```

Comment:
Hi! Great!
Do you want me to approve when you submit the rest of the translation?